pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -221,6 +221,10 @@ func (r *NetworkInfoReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		// AKO needs to know the AVI subnet path created by NSX
 		setVPCNetworkConfigurationStatusWithLBS(ctx, r.Client, ncName, state.Name, path, nsxLBSPath, *createdVpc.Path)
 		updateSuccess(r, ctx, obj, r.Client, state, nc.Name, path)
+		if !gatewayConnectionReady && ncName == commonservice.SystemVPCNetworkConfigurationName {
+			log.Info("requeuing the networkinfo because GatewayConnectionStatus is not ready", "req", req, "gatewayConnectionReason", gatewayConnectionReason)
+			return common.ResultRequeueAfter60sec, nil
```

Comment:
can we retry every 10 or 30 seconds?

Reply:
I prefer to keep it 60 seconds.. because all the namespaces binding to system vpcnetworkconfiguration will be requeued, e.g. kube-system, vmware-system-ako, vmware-system-monitoring, vmware-system-pinniped, vmware-system-cert-manager, ...  If we requeue these namespace too frequently, there will be too many invalid info in the NSX operator log..
And previously we set the non-system vpcs checking interval to 60s... https://github.com/vmware-tanzu/nsx-operator/pull/672#discussion_r1708669091